ajadriver/linux/ntv2devicefeatures.c

```diff
@@ -0,0 +1 @@
+/home/daniel/ntv2/ajalibraries/ajantv2/src/ntv2devicefeatures.cpp
```

Comment:
@Subsentient Did you intend to commit these paths that are specific to your system? I also see the same below in two other `*.c` files.

Reply:
Ahh, nah, sorry about that, those are apparently autogenerated symlinks, must have missed those. I'll fix it.

Reply:
Fixed, the makefile just regenerates those symlinks anyways so I just removed them.